### ISSUE TITLE
transport not passed to Email::Sender::Simple->sendmail properly

### ISSUE THREAD

**Issue by bigpresh** [gryphonshafer/Email-Mailer#1]
From trying to help someone on the beginners@perl.org mailing list having a problem with Email::Mailer ignoring a transport:
http://www.mail-archive.com/beginners%40perl.org/msg119395.html

... I think he found a bug in Email::Mailer.

Email::Mailer sends the email with:

```perl
    # send the email with Email::Sender::Simple
    sendmail( $email_mime, $mail->{transport} );
```

but looking at Email::Sender::Simple->send_email(), :

```perl
    sub send_email {
        my ($class, $email, $arg) = @_;
 
        my $transport = $class->default_transport;
 
        if ($arg->{transport}) {
            ...
```
So, Email::Sender::Simple->send_email() is expecting the second parameter passed to it to be a hashref containing a 'transport' key, but it's getting passed an Email::Sender::Transport::SMTP object
instead, and ignoring it.

**Comment by gryphonshafer** [comment]
Yes, looks like a bug. Should be trivial to fix, but it'll take me a few minutes to spin up an environment. I should have a new version released shortly thereafter.

**Comment by gryphonshafer** [comment]
OK, this should be resolved now under v1.05, just released. Should show up on CPAN in a few hours.

**Comment by bigpresh** [comment]
Fantastically quick response :)  Thanks!
